friendly_traceback/path_info.py

```diff
@@ -55,19 +64,19 @@ def exclude_directory_from_traceback(dir_name):
 exclude_directory_from_traceback(dirname)
 
 
-def is_excluded_file(full_path):
+def is_excluded_file(full_path: StrPath) -> bool:
     """Determines if the file belongs to the group that is excluded from tracebacks."""
-    if full_path.startswith("<frozen "):
-        return True
     # full_path could be a pathlib.Path instance
     full_path = str(full_path)
```

Comment:
Thinking out loud: maybe [`os.path.fspath`](https://docs.python.org/3/library/os.html#os.fspath) would be a safer converter. However, can be done in another issue as well.

Reply:
Interesting: I would not even have thought of this as a possible issue.

After a quick internet search, I gather that using str() will generally lead to more readable file paths (definitely desirable for users of friendly-traceback) but, for some applications, might lead with problems on systems where paths include non-utf8 characters.  I gather that the problems occur when an attempt is made to do the conversion in reverse.  Here, we always go from path to str(), and never in reverse.  Thus, I believe that it should be safe to convert to str, and get the benefit of nicer representation.